### ISSUE TITLE
Uncaught Error: Cannot find module './modules/ColorRegexes'

### ISSUE THREAD

**Issue by EnricoSignori** [thomaslindstrom/color-picker#109]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.204.0
**System**: Mac OS X 10.10.3
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v2.0.3

### Stack Trace

Uncaught Error: Cannot find module './modules/ColorRegexes'

```
At module.js:330

Error: Cannot find module './modules/ColorRegexes'
  at Module._resolveFilename (module.js:328:15)
  at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:270:25)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.getColorsInString (/Users/drgabas/.atom/packages/color-picker/lib/ColorPicker-view.coffee:252:13)
  at Object.open (/Users/drgabas/.atom/packages/color-picker/lib/ColorPicker-view.coffee:333:32)
  at atom-text-editor.<anonymous> (/Users/drgabas/.atom/packages/color-picker/lib/ColorPicker.coffee:42:68)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:178:20)

```

### Commands

```
     -8:19.5.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-mac-classic.theme-one-light-ui)
  2x -8:06.8.0 pane:show-item-1 (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-mac-classic.theme-one-light-ui)
     -8:02.6.0 pane:show-item-2 (ul.list-inline.tab-bar.inset-panel)
     -8:02.2.0 pane:show-item-3 (atom-text-editor.editor.is-focused)
     -8:01.9.0 pane:show-item-4 (atom-text-editor.editor.is-focused)
     -7:23.1.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-mac-classic.theme-one-light-ui)
  2x -7:20.9.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-mac-classic.theme-one-light-ui)
  5x -7:05.4.0 core:copy (div.deprecation-cop.pane-item.native-key-bindings)
  2x -0:06.6.0 color-picker:open (ul.list-inline.tab-bar.inset-panel)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "mac-classic"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.9
autocomplete-php, v0.3.6
color-picker, v2.0.3
file-icons, v1.5.6
highlight-selected, v0.9.3
linter, v0.12.6
linter-php, v0.0.15
mac-classic, v0.2.0
minimap, v4.9.2
save-session, v0.15.5

# Dev
No dev packages
```

**Comment by thomaslindstrom** [comment]
Hi, and thanks. This should be fixed in `v2.0.4`. https://github.com/thomaslindstrom/color-picker/commit/3f84d85bbe5c46c7f3c4abfafe23092a040d0f6a

**Comment by EnricoSignori** [comment]
ohh thanks for your attention.

2015-06-02 5:57 GMT-03:00 Thomas Lindstrøm <notifications@github.com>:
